src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int maxBound = 100;//range random 0...100
```

Comment:
```suggestion
    private static final int MAX_BOUND = 100;//range random 0...100
```
for naming constants SCREAMING_SNAKE_CASE is used